util/devel/lookForTabs

```diff
@@ -7,18 +7,31 @@
 
 cd $CHPL_HOME >& /dev/null
 
+passed=true 
 git grep -n $'\t' -- ':!*bison-chapel.cpp' ':!*bison-chapel.h' \
                      ':!*flex-chapel.cpp'  ':!*flex-chapel.h'  \
                      ':!*bison-chpl-lib.cpp'  ':!*bison-chpl-lib.h'  \
                      ':!*flex-chpl-lib.cpp'  ':!*flex-chpl-lib.h'  \
                      'compiler/*.lex'      'compiler/*.y'      \
                      'compiler/*.ypp'      'compiler/*.h'      \
                      'compiler/*.c'        'compiler/*.cpp'
-
+if [ $? -ne 1 ]; then 
+  passed=false
+fi  
 git grep -n $'\t' -- 'runtime/*.h' 'runtime/*.c' 'runtime/*.cpp'
-
+if [ $? -ne 1 ]; then 
+  passed=false
+fi  
 git grep -n $'\t' -- 'modules/*.chpl'
-
```

Comment:
I find this file hard to read/parse without some spacing here. I also think it'd be helpful to comment somewhere that a status code of 1 indicates "no matches were found" for grep since that wasn't intuitive to me.

Reply:
Added a  comment to indicate when the exit code will be 1.